roles/openshift_aws/tasks/setup_master_instances.yml

```diff
@@ -0,0 +1,38 @@
+---
```

Comment:
What is the difference between this file and setup_master_group.yml ?

Did you mean to rename setup_master_group.yml to this file?  setup_master_group.yml is used by cluster-operator plays.

Reply:
Right, but my question is, do we need this file at all?  Looks like we can just use setup_master_group.yml as-is instead of adding this file?